### DISCUSSION TITLE
TeX Live 2023 兼容性

### DISCUSSION THREAD

**Discussion by AyaTheCrow** [nju-lug/NJUThesis#203] [General]
感谢作者们的付出。

目前TexLive 2023已经发布，而且南大开源镜像站提供的版本亦为2023。请问该模板对新版本TexLive的兼容性如何？

**Comment by atxy-blip** [comment]
我第一时间更新了 TeX Live 2023，没有用出兼容问题。TL2023 的一个显著变化是 Windows 版终于提供了64位的程序，有一定性能提升，推荐升级试试。
